src/postgres/src/backend/parser/gram.y

```diff
@@ -975,9 +975,9 @@ stmt :
 
 			/* Not supported statements */
 			| AlterCollationStmt { parser_ybc_not_support(@1, "This statement"); }
-			| AlterFdwStmt { parser_ybc_not_support(@1, "This statement"); }
-			| AlterForeignServerStmt { parser_ybc_not_support(@1, "This statement"); }
-			| AlterForeignTableStmt { parser_ybc_not_support(@1, "This statement"); }
+			| AlterFdwStmt
```

Comment:
The new features from this list should go to the list of beta features.

Reply:
Yeah I believe the new features should be moved to that list now

Reply:
Moved all the features to the beta section: https://github.com/yugabyte/yugabyte-db/pull/9650/commits/940f02190359656406e4dd58f2b7c72d9fa9716d.